frontend/src/BoundedContext/UbiquitousLanguage.elm

```diff
@@ -139,7 +139,7 @@ languageTerms : UbiquitousLanguage -> List LanguageTerm
 languageTerms (UbiquitousLanguage terms) =
   terms
   |> Dict.toList
-  |> List.map (\(key, { term, description }) -> LanguageTerm (DomainTermId key) term description)
+  |> List.map (\(shortName, { term, description }) -> LanguageTerm (DomainTermId shortName) term description)
```

Comment:
While this rename is probably not necessary in the context of the task at-hand, it makes sense nevertheless.
For reference, this is the relevant screen:
![image](https://user-images.githubusercontent.com/616204/191185850-60651464-b811-4aca-84a8-929ff4b66654.png)